zulipterminal/config/keys.py

```diff
@@ -287,6 +287,16 @@ class KeyBinding(TypedDict):
         'excluded_from_random_tips': True,
         'key_category': 'msg_actions',
     },
+    'FULL_RENDERED_MESSAGE': {
+        'keys': ['f'],
+        'help_text': 'Show/hide full rendered message (from message information)',
+        'key_category': 'msg_actions',
+    },
+    'FULL_RAW_MESSAGE': {
+        'keys': ['r'],
+        'help_text': 'Show/hide full raw message (from message information)',
+        'key_category': 'msg_actions',
+    },
```

Comment:
I'm not averse to this part of the commit, but this isn't related to reordering the keys, based on the output in `hotkeys.md` being unchanged? That is, this seems like an independent refactor.